en/docs/develop/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -69,16 +69,20 @@ Ex: Enable landing page. **defaultTheme.json** ( Teant theming ).
                  └── favicon.ico
       ```
 
-3.  Sign in to the WSO2 Admin Portal ([https://server-host:9443/admin](https://server-host:9443/admin)) with your tenant username (format `<username>@<domain>.com kim@testorg.com`) and password.
+3.  Sign in to the WSO2 Admin Portal (`https://<server-host>:<server-port>/admin`) with your tenant username (format: `<username>@<domain>.com`) and password.
 
-4.  Expand the **Settings** menu, click **Upload Tenant Theme** and upload your ZIP file. 
+4.  Click **Tenant Theme** under **Settings** category and click **Browse File to Upload** to upload your ZIP file.
+    Alternatively, you can drag and drop your ZIP file to upload. 
 
     ![Upload tenant theme]({{base_path}}/assets/img/learn/upload-tenant-theme.png)
 
-5.  Access the API Developer Portal (`https://<server-host>:9443/devportal`) using your tenant username and password.
+5.  Access the API Developer Portal (`https://<server-host>:<server-port>/devportal`) using your tenant username and password.
 
     Note the new theme is applied.
     
+!!!Note
```

Comment:
If the UI is self explanatory, there's no point mentioning this in the docs.

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/1659/commits/04ddc59e90b732a370a7e4752bbbc7ac16f267f0